pkg/controller/controller_utils.go

```diff
@@ -772,6 +772,68 @@ func (s ActivePods) Less(i, j int) bool {
 	return false
 }
 
+// ActivePodsWithRanks is a sortable list of pods and a list of corresponding
```

Comment:
You need to describe the basic ordering in Godoc here for Less, and also add a godoc comment on Less

Reply:
Done.